cache/local_session.cpp

```diff
@@ -248,26 +248,25 @@ int local_session::write(const dnet_id &id,
 	return err;
 }
 
-std::unique_ptr<ioremap::elliptics::n2::lookup_response> local_session::lookup(const dnet_cmd &tmp_cmd, int *errp)
+std::shared_ptr<n2::lookup_response> local_session::lookup(const dnet_cmd &tmp_cmd, int *errp)
 {
-	std::unique_ptr<n2::lookup_request> request(new n2::lookup_request(tmp_cmd));
-	request->cmd.flags |= m_cflags;
-	request->cmd.size = 0;
-	request->cmd.backend_id = m_backend.backend_id();
+	n2_request request(tmp_cmd, n2::default_deadline());
```

Comment:
Why is it uses default deadline?

Reply:
We should specify deadline for each request. In previous master-branch lookup request hadn't deadline in its body, so I think we can specify some default value. Doesn't it?

Reply:
Will be fixed later: https://st.yandex-team.ru/ELL-924

TODO is put.